tests/fixtures/smoke-tests/passing-test/tests/acceptance/acceptance-test.js

```diff
@@ -0,0 +1,18 @@
+import moduleForAcceptance from '../helpers/module-for-acceptance';
+import { test } from 'qunit';
+
+let beforeEachWorks;
+
+moduleForAcceptance('Module', {
+  beforeEach() {
+    beforeEachWorks = true;
```

Comment:
Yeah, you can assert that `this.application` exists (since that is done inside `moduleForAcceptance`), and that the first argument is the QUnit assert object (`Object.getPrototypeOf(arguments[0]) === QUnit.assert` *might* work).